daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/TransactionVersion.scala

```diff
@@ -50,6 +50,7 @@ object TransactionVersion {
   private[lf] val minExceptions = V14
   private[lf] val minByKey = V14
   private[lf] val minInterfaces = VDev
+  private[lf] val minExplicitDisclosure = VDev
```

Comment:
This value is currently used by testing when building versioned contract instances